titanic.ipynb

```diff
@@ -8,6 +8,7 @@
         "## Titanic 3\n",
```

Comment:
_Commented on notebook `titanic.ipynb` Cell 2 Line 6_
```python
import seaborn as sns
import matplotlib.pyplot as plt
import matplotlib.pyplot as plt

# Load the Titanic dataset
titanic = sns.load_dataset('titanic')
```
A single comment on titanic

_Respond and view the context [here](https://gitnotebooks.com/smith-kyle/notebook-diff-tests/pull/10)._

Reply:
reply to a single comment

Reply:
reply to a reply to a single comment made from github